ethereum_history_api/circuits/lib/src/merkle_patricia_proofs/proof.nr

```diff
@@ -1,11 +1,15 @@
 use crate::rlp::decode::{decode_to_rlp_header, decode_list_of_small_strings, RlpList};
 use crate::misc::{
     fragment::Fragment, arrays::{sub_array_equals_up_to_length, memcpy_up_to_length}, types::Bytes32,
-    bytes::{bytes_to_nibbles, right_pad}
+    bytes::{bytes_to_nibbles, byte_to_nibbles, right_pad}
 };
 use dep::std::hash::keccak256;
 
 global MAX_NODE_LEN = 532;
+global MAX_NODE_FIELDS = 17;
+global BRANCH_NODE = 17;
+global EXTENSION_NODE = 2;
+global HASH_LEN = 32;
```

Comment:
Use one from lib

Reply:
And move to:
https://github.com/vlayer-xyz/noir-ethereum-history-api/blob/b7ec975214a9837cb45dde2eb9a64548131824c3/ethereum_history_api/circuits/lib/src/misc/types.nr#L4